tests/core/test_int53.c

```diff
@@ -0,0 +1,412 @@
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
+#endif
+#include <stdint.h>
+#include <stdio.h>
+#include <float.h>
+#include <math.h>
+
+#include <time.h>
+#include <stdlib.h>
+
+// Uncomment to compute the expected result:
+//#define ANSWERS
+
+void writeI53ToI64_int64(int64_t *heapAddress, int64_t num)
+{
+#ifdef ANSWERS
+	*heapAddress = num;
+#else
+	EM_ASM(writeI53ToI64($0, $1), heapAddress, (double)num);
+#endif
```

Comment:
Would it make sense somehow do both of these at runtime compare the results?  (Rather than have a compile time switch?)

Reply:
The results are compared to the originals in the body of the loop below.